### PR TITLE
Clean up old call to byName

### PR DESCRIPTION
**Pull request by peckto**

### CODE REVIEW

**Review by oxisto — APPROVED** [review]

File: cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt [hunk 1]
```diff
@@ -59,7 +59,6 @@ class CompilationDatabase : Plugin {
                     "import de.fraunhofer.aisec.cpg.graph.allChildren",
                     "import de.fraunhofer.aisec.cpg.graph.ast",
```
Comment (oxisto) [inline cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: I think we need to include `import de.fraunhofer.aisec.cpg.graph.*` here I think  and it looks like something is still using `byName`.

Comment (peckto) [reply cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: Its already there:
https://github.com/Fraunhofer-AISEC/cpg/pull/1618/files#diff-2ceb1018c52a5beec418b3ab986e10abe7a5ebda5469c0900af841e9b0ef050aL53

Comment (peckto) [reply cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: Does `byName` still exist? I cant find the symbol anymore.

Comment (oxisto) [reply cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: Not sure, but the test fails now :D

Comment (oxisto) [reply cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: Ah! you removed one line and the test checks the number of lines. So we need to adjust the assert from 19 to 18.

Comment (peckto) [reply cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt:60]: ah, should be fixed now

**Pull request merged**